clientSide/save_cow/src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,42 @@
+.footer-container
+{
+    height: 400px;
+    width: 100%;
+    background-color: navy;
+    border-radius: 150px 0px 0px 0px;
+    display: flex;
+}
+.savecow{
+    margin : 50px 0px 0px 200px;
+    color: white;
+}
+.footer-wrapper
+{
+    color: white;
+    margin: 50px 0px 0px 150px;
+    line-height: 30px;
+    font-size: 20px;
+}
+.footer-social > p{
+    color: white;
+    margin: 0px 0px 0px 50px;
+    font-size: 20px;
+}
+.hospital{
```

Comment:
please don't repeat any repeated CSS properties 

create one class name and use same class name for remaining